src/Css.elm

```diff
@@ -598,8 +665,8 @@ type Value supports
 
 {-| A type used to specify which properties and which values work together.
 -}
-type Supported
-    = Supported
+type alias Supported =
+    Never
```

Comment:
Little proposition for making phantom types more phantom :) Like never before.

Reply:
Actually there are more variants:

1. If we don't want that user will describe phantom types of `Value` with both `Never` and `Supported`
	```elm
	type Supported
		= Supported Never
	```
2. We may not use any special type, just empty tuple. Error messages keep same.
	```elm
	dotted : Value { provides | dotted : () }
	dotted =
	    Value "dotted"
	```

Reply:
A nice feature of having `Supported` be a union type that doesn't expose its constructor is that other modules can't instantiate it. This limits how other modules can use it, which in this case is desirable!

On the other hand, `Never` and `()` can be instantiated outside this module (by using `Basics.never` and `()`, respectively), which means we're potentially giving up guarantees around how these values can be constructed, without necessarily knowing what those guarantees are are or why we're giving them up. 😅

To make a long story short, I definitely want to leave this as-is! 😄